### PR TITLE
(DOCSP-25554): Export favorite connections with the CLI

### PR DESCRIPTION
**Pull request by jeff-allen-mongo**
### DESCRIPTION

This PR adds these pages:

- Landing page for CLI import / export
- Page for exporting a favorite connection (unencrypted)

Importing and encrypting are coming in future PRs.

### JIRA

[DOCSP-25554](https://jira.mongodb.org/browse/DOCSP-25554 ) (part of [DOCSP-25219](https://jira.mongodb.org/browse/DOCSP-25219))

### STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-25554/connect/favorite-connections/import-export-cli/

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-25554/connect/favorite-connections/import-export-cli/export/#std-label-compass-export-connections-cli

### CODE REVIEW

File: source/connect/favorite-connections/import-export-cli/export.txt [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+By default, |compass-short| does not hide sensitive connection
+information (such as usernames and passwords) from the exported
+connection file. To learn how to protect sensitive information in
+exported connections, see :ref:`compass-encrypt-connections-cli`.
+
+Procedure
+---------
+
+To export **unencrypted** favorite connections using the |compass-short|
```
Comment (davemungo) [inline source/connect/favorite-connections/import-export-cli/export.txt:33]: Consider:

... connections with the ...

File: source/connect/favorite-connections/import-export-cli/export.txt [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+.. code-block:: sh
+
+   <path-to-Compass-executable> --export-connections=<filename>
+
+Example
+-------
+
+This example exports favorite |compass-short| connections to a file with
+the path ``/tmp/compass-connections/favorites.json``. Run the following
+command in the folder containing your MongoDB Compass binary:
```
Comment (davemungo) [inline source/connect/favorite-connections/import-export-cli/export.txt:53]: Should this say something like, 

"On MacOS the command looks like the following:"
or 
"Each operating system is slightly different, on MacOS the command looks like the following:"

**Review by davemungo — COMMENTED** [review]
@jeff-allen-mongo Thanks Jeff, looking good so far. I left a couple of suggestions.

Is this work going into a branch? It looks like it is. If not, consider commenting out references to the unfinished pages in case it gets pushed by mistake.

File: source/connect/favorite-connections/import-export-cli.txt [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can use the |compass-short| :abbr:`CLI (Command-Line Interface)` to
+export and import favorite connections. This lets you share saved
+connections between workspaces and with colleagues to quickly connect to
+shared deployments.
+
+By default, |compass-short| does not hide sensitive connection
```
[the remark this answers is not in the document]
Comment (jeff-allen-mongo) [reply source/connect/favorite-connections/import-export-cli.txt]: Updated to an include.

File: source/connect/favorite-connections/import-export-cli/export.txt (hunk 1 above)
Comment (jeff-allen-mongo) [reply source/connect/favorite-connections/import-export-cli/export.txt]: Done.

File: source/connect/favorite-connections/import-export-cli/export.txt [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+connection file. To learn how to protect sensitive information in
+exported connections, see :ref:`compass-encrypt-connections-cli`.
+
+Procedure
+---------
+
+To export **unencrypted** favorite connections using the |compass-short|
+:abbr:`CLI (Command-Line Interface)`, specify:
+
+- The path to the |compass| executable. The path to the executable
```
[the remark this answers is not in the document]
Comment (jeff-allen-mongo) [reply source/connect/favorite-connections/import-export-cli/export.txt]: Done.

**Review by jeff-allen-mongo — COMMENTED** [review]
@davemungo, thanks for the review. Could you take another look?

Jeff

**Comment by jeff-allen-mongo** [comment]
To answer your previous question, this is being merged into a feature branch, so no risk of going live with empty pages.

File: source/includes/export-connection-warning.rst [hunk 5]
```diff
@@ -0,0 +1,3 @@
+When you export favorite connections, database usernames and
+passwords are included in plaintext unless you :ref:`encrypt the
```
Comment (davemungo) [inline source/includes/export-connection-warning.rst:2]: This renders properly, but 'encrypt the' has a strange underline in GitHub. Maybe just a GH artifact?

**Review by davemungo — APPROVED** [review]
@jeff-allen-mongo Thanks for the updates, this LGTM

File: source/connect/favorite-connections/import-export-cli/export.txt [hunk 6]
```diff
@@ -0,0 +1,124 @@
+.. _compass-export-connections-cli:
+
+========================================
+Export Favorite Connections with the CLI
```
Comment (Anemy) [inline source/connect/favorite-connections/import-export-cli/export.txt:4]: Exporting encrypted connections is very similar to exporting connections unencrypted. The only difference being the used of the `--passphrase=<passphrase>` command line argument.
I think it's good if we guide folks towards using a passphrase if possible.
How do you feel about making the default landing page for `Export Favorite Connections with the CLI` instead show the way to export with a passphrase? And then we can add an extra section about not using a passphrase to export the connections as unencrypted. That way we can avoid having an extra page for that as well?

**Comment by jeff-allen-mongo** [comment]
Hey @Anemy, I restructured this a bit and put the procedures for exporting both encrypted and unencrypted connections on the same page. I have the encrypted procedure first so users should see this as the "default". When you have a moment, could you take a look and see if this works?

Thanks

**Review by Anemy — APPROVED** [review]
lgtm!

**Pull request merged**